src/main/resources/tree.json

```
@@ -78,9 +73,101 @@
     },
     {
       "key": "EXCEPT_M1",
```

```suggestion
      "key": "OTHER",
```

---

Let's disscuss it on a call

---

OK, let's discuss. In my opinion "other" is too wide - we can use it everywhere, but "except m1" is rather narrow and when I see "except m1" I clearly understand, that it is any type of vehicle except passenger auto. Probably, we can use "m1_vehicles" and "other_vehicles"?

---

Please, add description